src/apof/baskets/tests.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from mock import Mock
+
+from baskets.models import Basket, Order
+from menus.models import Meal
+
+
+class BasketTest(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create_user(
```

Comment:
It seems like this user creation is in every unit test that requires user.
Please move it to fixture and update every test that is using it.